css/templates/e-commerce/page.css

```diff
@@ -0,0 +1,216 @@
+@import '../../core';
+@import 'page-vars';
+@import '../../../components/inputs/inputs';
+@import '../../../components/social-follow/social-follow';
+
+body {
+  font-size: var(--narrow-body-font-size);
+}
+
+@media (--breakpoint-md) {
+  body {
+    font-size: var(--wide-body-font-size);
+  }
+}
+
+main {
+  max-width: none;
+}
+
+
+/* TYPOGRAPHY */
+.h3 {
+  color: var(--color-font-title);
+  line-height: normal;
+  padding-bottom: var(--space-1);
+}
+
+
+/* ELEMENTS */
+.ampstart-btn-secondary {
+  background-color: var(--color-white);
+  color: var(--color-primary);
+  border-color: var(--color-primary);
+  font-size: var(--button-font-size);
+  font-family: var(--font-family);
+  font-weight: var(--bold-font-weight);
+}
+
+.ampstart-hero .ampstart-btn {
+  padding: var(--space-1) var(--space-3);
+}
+
+
+/* HEADER */
+.ampstart-headerbar {
+  color: var(--color-primary);
+  background-color: var(--color-white);
+  height: var(--narrow-nav-height);
+  z-index: var(--z1);
+}
+
+.ampstart-navbar-trigger {
+  cursor: pointer;
```

Comment:
It does, but I'm not including that CSS since it contains many styles that I'd need to override.